tests/functional/test_deploys_nixos/__init__.py

```diff
@@ -0,0 +1,69 @@
+from nixops.util import root_dir
+from itertools import product
+from parameterized import parameterized
+
+from tests.functional.shared.deployment_run_command import deployment_run_command
+from tests.functional.shared.create_deployment import create_deployment
+from tests.functional.shared.using_unique_state_file import using_unique_state_file
+
+@parameterized(product(
```

Comment:
well, since `@parameterized` is used in lots of places in tests, I think we can document it in `doc/hacking.xml`

@coretemp , what do you think?

P.S. @parametrized is from here https://github.com/wolever/parameterized